pysat/tests/test_utils.py

```diff
@@ -61,7 +86,7 @@ def teardown(self):
         del self.ci_env
```

Comment:
setup and teardown docstrings should be in imperative form

Reply:
I am confused by this statement and don't know how to address it.

Reply:
Changing "Runs" to "Run" fixes the problem (can't implement as a suggestion here).  Elsewhere, "Set up the unit test environment." and "Clean up the unit test environment." have been used, which I think is a little cleaner.  This is endemic throughout unit tests though, so maybe a full change is out of scope.